### PR TITLE
fighting limbo state

### PR DESCRIPTION
**Pull request by hthayko**
should resolve the limbo state. Added catch() to getLatestRow()

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -29,14 +29,16 @@ app.get('/', function(req, res) {
...
+  .then(function(row) {
     var rippleds = rc_util.getRippledsC(JSON.parse(row.data));
     var flatRippleds = _.map(rippleds, function(item, key) {
       item.public_key = key;
       return item;
     });
     res.send(flatRippleds);
-  });
+  })
+  .catch(res.send);
```
Comment (wilsonianb) [inline server.js]: This will return the error in the api response?

Comment (hthayko) [inline server.js]: yeah. that's a bad idea right ?

Comment (wilsonianb) [inline server.js]: maybe log the error and respond with `500`.

Comment (hthayko) [inline server.js]: OK

**Comment by wilsonianb** [comment]
LGTM :+1:

**Pull request merged**